CODE/Sources/PennyBOT/Penny.swift

```diff
@@ -52,4 +48,42 @@ struct Penny {
         
         RunLoop.current.run()
     }
+    
+    static func bootstrapLoggingSystem(httpClient: HTTPClient) {
+        guard let webhookUrl = Constants.loggingWebhookUrl,
+              let token = Constants.botToken
+        else {
+            fatalError("Missing 'LOGGING_WEBHOOK_URL' or 'BOT_TOKEN' env vars")
+        }
+        DiscordGlobalConfiguration.logManager = DiscordLogManager(
+            client: DefaultDiscordClient(
+                httpClient: httpClient,
+                token: token,
+                appId: nil
+            ),
+            configuration: .init(
+                fallbackLogger: Logger(
```

Comment:
If there is a problem with sending logs to Discord, the problems will be reported using this logger.